app/services/pii/cacher.rb

```diff
@@ -30,6 +30,19 @@ def fetch
       Pii::Attributes.new_from_json(pii_string)
     end
 
+    def exists_in_session?
```

Comment:
This is an annoying diff. The actual change was a copy and past of the `fetch_string` method.